common/test/java/com/couchbase/lite/LiveQueryTest.java

```diff
@@ -129,21 +129,16 @@ public void testLiveQueryDelay() throws CouchbaseLiteException, InterruptedExcep
             });
 
         try {
-
-            // give it a few ms to deliver the first notification
-            Thread.sleep(SLOP_MS);
-
             createDocNumbered(12);
             createDocNumbered(13);
             createDocNumbered(14);
             createDocNumbered(15);
             createDocNumbered(16);
 
-            Thread.sleep(LiveQuery.UPDATE_INTERVAL_MS + SLOP_MS);
+            Thread.sleep(AbstractQuery.UPDATE_DELAY_MS + SLOP_MS); //sleep for 200ms and a bit more
 
-            assertEquals(3, times[0]);
-            assertTrue(times[2] - times[1] < LiveQuery.UPDATE_INTERVAL_MS);
```

Comment:
Can you explain what this test is testing, now?  I'm really unclear on what it does.

Reply:
This test is supposed to test that the time change is posted is at least 200ms away from when we start the query since we need that delay to complete registering observer.